src/index.ts

```diff
@@ -8,7 +8,11 @@ import * as yargs from 'yargs'
 import workerpool from 'workerpool'
 import path from 'path'
 import os from 'os'
-import { register, buyAmountEstimationMetrics, orderBookMetrics } from './metrics'
+import { buyAmountEstimationMetrics, orderBookMetrics } from './metrics'
+import promBundle from 'express-prom-bundle'
+
+const BASE_BATH = process.env.BASE_BATH || '/api/v1'
```

Comment:
nit: For consistency, I would prefer if this was also yargs options, that way it can be specified both as a command line argument or as an environment variable like the other configuration options.

Reply:
Changed in e938285eb5b0ba74cb2ce933e0db0638f2b22b47